erpnext/patches/v12_0/delete_report_requested_items_to_order.py

```diff
@@ -0,0 +1,7 @@
+import frappe
+
+def execute():
+	frappe.db.sql("""
+		DELETE FROM `tabReport`
+		WHERE name = 'Requested Items to Order'
+	""")
```

Comment:
Need to delete auto email reports associated with this too. Refer to erpnext/patches/v13_0/delete_old_purchase_reports.py